190925

mapcat opened the pull request.

yakra commented:
BOOM! Looks like this is safe to merge into mapcat:master, should you choose to do so.

mapcat commented:
Thanks. I'll take care of moving the usansf routes.

Markkos1992 committed:
PA 8: Recentering Along PA 130

Markkos1992 committed:
I-376: Lining Up with PA 130

Markkos1992 committed:
US 22: Lining Up with PA 130

Markkos1992 committed:
PA 993: Recentering

Added Baker School Rd (BakSchRd), Oak St (OakSt_S), Sandy Hill Rd (SanHillRd), Bushy Run Battlefield Park (BusRunBP), and Claridge Elliott Rd (ClaEliRd).  Replaced +X1 with Irwin St (IrwSt).  Replaced FrogRd with Brownstown Rd (BroRd).  Replaced +X4 with Ridge Rd  (RidRd).

Markkos1992 committed:
PA 66 BUS (Greensburg): Added Points

Added Oakford Park Rd (OakParkRd) and Old Route 66 (OldRt66).

Markkos1992 committed:
US 22: Added Points

Added Berlin Farm Rd (BerFarmRd) and Mellon Rd (MelRd).

Markkos1992 committed:
PA 130: Recentering

Added+X1R, Nadine Rd (NadRd), Verona Rd (VerRd), Homer Dr (HomDr), Thompson St (ThoSt), Forbes Rd (ForRd_W and ForRd_E), Burrell Hill Rd (BurHillRd), Greengate Rd (GreRd), Myers Rd (MyeRd), Marguerite Lake Rd (MarLakeRd), Possum Hollow Rd (PosHolRd), +X4, Dillon Rd (DilRd), Austin Blvd (AusBlvd), and +X5.   Marked Wall Bridge (*WallBri) as closed).  Relocated +X1.  Replaced +X4 with Paintertown Rd (PaiRd).  Replaced WelSt (in-use) with Altman Rd (AltRd).

Markkos1992 committed:
US 30: Added Points 

Added Ardana Rd (ArdRd), Carpenter Ln (CarLn), Skellytown Rd (SkeRd), Center Hwy (CenHwy), Fairwood Dr (FaiDr), Barnes Lake Rd (BarLakeRd), Lowry Ave (LowAve), Greengate Rd  (GreRd_A), and +X01AB.

Markkos1992 committed:
PA 136: Added Edna Rd.

Markkos1992 committed:
PA 982: Added Alice Rd (AliRd) and Possum Hollow Rd (PosHolRd)

michihdeu committed:
HE fix

michihdeu committed:
Merge remote-tracking branch 'refs/remotes/TravelMapping/master'

michihdeu committed:
GES D8 + D20-D29 (54)

michihdeu committed:
Merge pull request #3171 from michihdeu/master

GES/HE

yakra committed:
NS: Marshalltown bypass

yakra committed:
NY point label fixes

the-spui-ninja committed:
Merge pull request #4 from TravelMapping/master

sretdgyuhij

the-spui-ninja committed:
Update nd.i094.wpt

yakra committed:
Nebraska directional prefixes (numeric)

yakra committed:
Kansas directional prefixes (numeric)

yakra committed:
Oklahoma directional prefixes (numeric)

jteresco committed:
Merge pull request #3169 from mapcat/master

KY: Extend KY 158 & delete KY 3303

jteresco committed:
Merge pull request #3170 from Markkos1992/PA

PA 130 and PA 993 Recentering

jteresco committed:
Merge pull request #3172 from yakra/Canada

NS: Marshalltown bypass

jteresco committed:
Merge pull request #3173 from yakra/RobertMoses

NY point label fixes

jteresco committed:
Merge pull request #3174 from the-spui-ninja/master

fix typo from last pull because I was rushed

yakra committed:
TX US77 concurrency fix

rickmastfan67 committed:
on.on401: corrected exit number

407 -> 406

rickmastfan67 committed:
on.on417: New exit 109

rickmastfan67 committed:
on.tchott: Added new ON-417 exit 109

http://forum.travelmapping.net/index.php?topic=3222.0

rickmastfan67 committed:
canon: update ferry labels on ON-6

http://forum.travelmapping.net/index.php?topic=3219.0

the-spui-ninja committed:
Merge pull request #5 from TravelMapping/master

Prendo un cornetto e un caffé macchiato

the-spui-ninja committed:
Update mt.i090bllau.wpt

the-spui-ninja committed:
Update mt.sr532.wpt

the-spui-ninja committed:
Update mt.us310.wpt

the-spui-ninja committed:
Update updates.csv

rickmastfan67 committed:
cannf: ON updates

rickmastfan67 committed:
on.ondvp: DVP now has posted exit numbers

si404 committed:
datacheck

si404 committed:
Merge remote-tracking branch 'refs/remotes/TravelMapping/master'

si404 committed:
US10 Historic WA

si404 committed:
espga review A Coruna & Lugo

si404 committed:
sync

si404 committed:
espga review ourense & pontevedra

jteresco committed:
Merge pull request #3175 from yakra/CentralStates

Operation Directional Prefix (numeric)

jteresco committed:
Merge pull request #3176 from yakra/LoneStar

TX US77 concurrency fix

jteresco committed:
Merge pull request #3177 from the-spui-ninja/master

US 310 extension per forum

jteresco committed:
Merge pull request #3178 from TravelMapping/rickmastfan67-ON

Ontario updates

si404 committed:
Merge remote-tracking branch 'refs/remotes/TravelMapping/master'

michihdeu committed:
Merge remote-tracking branch 'refs/remotes/TravelMapping/master'

michihdeu committed:
Merge remote-tracking branch 'refs/remotes/TravelMapping/master'

michihdeu committed:
CZE point requests

michihdeu committed:
Merge pull request #3179 from michihdeu/master

CZE

si404 committed:
usaush

si404 committed:
datacheck fixes

jteresco committed:
Merge pull request #3180 from si404/master

espga and usaush
